atlp-rwanda/e-comm-team-brogrammers-fn
# #184948175 A buyer should  perform crud operations on  cart

#### What does this PR do?
- A buyer should be able to perform crud operations on cart and also fixed some bugs
#### Description of Task to be completed?
- A buyer should be able to perform crud operations on cart
#### How should this be manually tested?
after cloning this repository
1. add environment variables as are in .env.example
2. run npm install to install all dependencies
3. run npm start and open in it the browser
4. after `login` as a buyer
5. you'll find the `cart icon` button then jump to it to check your cart
6. you can add a product to cart using the cart icon
####  Bugs  fixed
-  added cursor-pointer when hovering on categories
-  position of the toasts should be consistent through out
-  add a label of expiration data field when adding a product
-  price of a product should be reflecting what you have input

#### Any background context you want to provide?
- n/a
#### What are the relevant pivotal tracker/Trello stories?
- [Add To Cart](https://www.pivotaltracker.com/n/projects/2639400/stories/184948175)
- [View Cart](https://www.pivotaltracker.com/n/projects/2639400/stories/184948176)
- [Clear Cart](https://www.pivotaltracker.com/n/projects/2639400/stories/184948177)
- [Update Cart](https://www.pivotaltracker.com/n/projects/2639400/stories/184948178)

#### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2023-05-19 at 13 30 40" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/4d69254c-0680-4358-826b-ce276a4c21d7">
<img width="1440" alt="Screenshot 2023-05-19 at 13 30 47" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/8ccdcba5-bbca-492d-a361-18e1e0679bb0">
<img width="1440" alt="Screenshot 2023-05-19 at 13 30 59" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/d2470295-5f43-4c00-a3f1-20f5d59af350">
<img width="1440" alt="Screenshot 2023-05-05 at 12 29 11" src="https://user-images.githubusercontent.com
<img width="1440" alt="Screenshot 2023-05-19 at 13 25 03" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/1248103e-8ada-4d16-9fac-e1536101f55c">
<img width="1440" alt="Screenshot 2023-05-19 at 13 25 13" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/fa49998b-65d6-418f-abe6-f16bec9b6f55">
<img width="1440" alt="Screenshot 2023-05-19 at 13 25 22" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/dd4fd1ef-40de-4cbc-916d-a5d797f1053c">
/56297754/236435167-43585a67-de57-4b6f-beaa-ea32ccd865f4.png">
<img width="1440" alt="Screenshot 2023-05-19 at 15 25 35" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/56297754/7b04def9-3f12-45a3-ade1-9835b62ce8e8">



#### Questions:
